pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -1636,11 +1632,6 @@ func parseConfigItems(config *zconfig.EdgeDevConfig, ctx *getconfigContext) {
 				"MetricInterval", oldMetricInterval, newMetricInterval)
 			updateMetricsTimer(newMetricInterval, ctx.metricsTickerHandle)
 		}
-		if newSSHAuthorizedKeys != oldSSHAuthorizedKeys {
```

Comment:
Don't you still need this code to handle modifications to the key?

Reply:
This is duplication of what is there inside NIM.

Reply:
Oh right.